bool code_ready    # 是否准备好执行动作
string action      # 执行的动作，例如 "up", "down", "reset"
